test/integration/local/test_multi_model.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import json
+import os
+import subprocess
+import sys
+import time
+
+import pytest
+import requests
+
+PING_URL = 'http://0.0.0.0:8080/ping'
```

Comment:
[Very Minor] Might be worth separating this variable out into pieces, in case the port changes. Maybe something like:
BASE_URL = 'http://0.0.0.0:8080/
PING_URL = BASE_URL + 'ping'
etc..